lib/autokey/scripting/engine.py

```diff
@@ -203,45 +203,32 @@ def create_phrase(self, folder, name: str, contents: str,
                                     not persisted as .jsons, and are replaced if the description is not
                                     unique within the folder.
                                      Used for single-source rc-style scripts.
+        @param replaceExistingHotkey: If true, instead of warning if the hotkey
```

Comment:
Don’t forget to update the doc-string

Reply:
Good point. If my refactoring tool wasn't broken (meaning I had to find/replace) I would have missed this